src/themes/color.rs

```diff
@@ -1,19 +1,193 @@
 use crate::errors::*;
-use color_space::{Hsv, Rgb};
 use serde::de::{self, Deserializer, Visitor};
 use serde::{Deserialize, Serialize, Serializer};
 use smart_default::SmartDefault;
 use std::fmt;
 use std::ops::Add;
 use std::str::FromStr;
 
+/// An RGBA color (red, green, blue, alpha).
+#[derive(Copy, Clone, Debug, Default, Eq, PartialEq)]
+pub struct Rgba {
+    pub r: u8,
+    pub g: u8,
+    pub b: u8,
+    pub a: u8,
+}
+
+impl Rgba {
+    /// Create a new RGBA color.
+    ///
+    /// `r`: red component (0 to 255).
+    ///
+    /// `g`: green component (0 to 255).
+    ///
+    /// `b`: blue component (0 to 255).
+    ///
+    /// `a`: alpha component (0 to 255).
+    #[inline]
+    pub fn new(r: u8, g: u8, b: u8, a: u8) -> Self {
+        Self { r, g, b, a }
+    }
+
+    /// Create a new RGBA color from the `hex` value.
+    ///
+    /// ```let cyan = Rgba::from_hex(0xffffff);```
+    pub fn from_hex(hex: u32) -> Self {
+        Self {
+            r: (hex >> 24) as u8,
+            g: (hex >> 16) as u8,
+            b: (hex >> 8) as u8,
+            a: hex as u8,
+        }
```

Comment:
```suggestion
        let [r, g, b, a] = hex.to_be_bytes();
        Self { r, g, b, a }
```

Reply:
@MaxVerevkin, that looks good. I'll amend my commit.